src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1019,6 +1023,34 @@ export class ResultsViewPageStore {
         }
     });
 
+    readonly areSequencedSamplesDifferentPerGene = remoteData<boolean>({
```

Comment:
why is this check needed? Why not let the tooltip always appear?

Reply:
I thought to only put that, along with the asterisk, if it's needed. What do you think? Always tooltip? Always asterisk? Always both?

Reply:
I think we can always display the tooltip.

Reply:
what about the asterisk?

Reply:
I think asterisk should only be there when there is `not profiled` samples in the row.

Reply:
regardless of whether the not profiled samples are different in each track, i.e. the denominator is the same in each track

Reply:
Yes, I think so. It's a useful feature to indicate there are `not profiled` samples in the row (or all rows) since they are at the end of the oncoprint.

Reply:
kk, that simplifies things